components/micro-gateway-core/src/main/ballerina/src/gateway/constants/api_security_constants.bal

```diff
@@ -151,7 +151,7 @@ public function getFailureMessageDetailDescription(int errorCode, string errorMe
         errorDescription += DESCRIPTION_SEPARATOR + API_AUTH_ACCESS_TOKEN_INACTIVE_DESCRIPTION;
     } else if (API_AUTH_MISSING_CREDENTIALS == errorCode) {
         string authHeaderName = getAuthorizationHeader(runtime:getInvocationContext());
-        errorDescription += DESCRIPTION_SEPARATOR + "Make sure your API invocation call has a header: \"" + authHeaderName + "\"";
+        errorDescription += DESCRIPTION_SEPARATOR + API_AUTH_MISSING_CREDENTIALS_DESCRIPTION ;
```

Comment:
Try to use the APIM message for the consistancy

Reply:
APIM message is "Make sure your API invocation call has a header". I believe this message is kind of invalid now.